components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/openidconnect/OpenIDConnectClaimFilterImplTest.java

```diff
@@ -0,0 +1,339 @@
+/*
+ *
+ *   Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
check the formatting of these license headers

Reply:
Addressed with f9303a8592add063b8b064ee19c83d907911296f